great_expectations/data_context/data_context.py

```diff
@@ -1718,19 +1716,29 @@ def _load_project_config(self):
 
     def _save_project_config(self):
         """Save the current project to disk."""
-        with open(os.path.join(self.root_directory, "great_expectations.yml"), "w") as data:
-            #FIXME: This method is currently Deactivated
-            print(type(self._project_config))
-            config = self._project_config.as_dict()
+        logger.debug("Starting DataContext._save_project_config")
+
+        config_filepath = os.path.join(self.root_directory, "great_expectations.yml")
+        with open(config_filepath, "w") as data:
+            #Note: I don't know how this method preserves commenting, but it seems to work
```

Comment:
This preserves commenting because the config is read in as a commentedmap initially by ruamel.yaml

But the if statement here should be removed...both of these paths re doing the same thing...

Reply:
Good point. Not sure how I missed that. Will change.